test/e2e/apimachinery/storage_version.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apimachinery
+
+import (
+	"context"
+	"time"
+
+	apiserverinternalv1alpha1 "k8s.io/api/apiserverinternal/v1alpha1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	"github.com/onsi/ginkgo"
+)
+
+const (
+	svName     = "storageversion.e2e.test.foos"
+	idNonExist = "id-non-exist"
+)
+
+var _ = SIGDescribe("StorageVersion resources [Feature:StorageVersionAPI]", func() {
```

Comment:
The alpha PR/periodic job config will need to opt into this test. See config in https://github.com/kubernetes/test-infra/ for:
* pull-kubernetes-e2e-gce-alpha-features
* ci-kubernetes-e2e-gci-gce-alpha-enabled-default
* ci-kubernetes-e2e-gci-gce-alpha-features


Once that's updated, we can do `/test pull-kubernetes-e2e-gce-alpha-features` on this PR and see a passing run of the test before merge.

Reply:
nice catch! I tested locally and didn't realize that. Opened https://github.com/kubernetes/test-infra/pull/20309